Sources/NIOCore/ChannelHandlers.swift

```diff
@@ -105,6 +105,11 @@ public final class AcceptBackoffHandler: ChannelDuplexHandler, RemovableChannelH
     }
 }
 
+#if swift(>=5.7)
```

Comment:
Actually a good question! 
In theory this syntax is already available since Swift 5.5. However, if you, for whatever reason, used `AcceptBackoffHandler` in way that does not pass sendebility checking, you would get an error and not a warning. In Swift 5.6 you will only get a warning in most cases, so we could also replace `swift(>=5.7)` with `swift(>=5.6)`. Nevertheless I have chosen 5.7 just to be a bit more on the safer side as the sensibility rules are in general even a bit more "relaxed" in more complex scenarios.
That being said, I'm still open to already add this in Swift 5.6 as I'm currently failing to create a code snippet that actually produces an error in 5.6.